src/aks-preview/azext_aks_preview/_params.py

```diff
@@ -225,6 +226,7 @@ def load_arguments(self, _):
             c.argument('kubelet_config', type=str)
             c.argument('linux_os_config', type=str)
             c.argument('enable_encryption_at_host', options_list=['--enable-encryption-at-host'], action='store_true')
+            c.argument('enable_ultra_ssd', action='store_true')
```

Comment:
Do we need to consider adding `is_preview=True` for this parameter?

Reply:
no, we will GA this feature this month.
And actually cli-extension is preview command, so it does not matter.

Reply:
OK, got it